core/src/main/java/io/ddf/DDF.java

```diff
@@ -346,6 +346,12 @@ private void validateSchema(Schema schema) throws DDFException {
         if (columnSet.contains(column.getName())) {
```

Comment:
Should we make this validation a Schema's method so that we can create a unit test for it without the need to instantiate any DDF?